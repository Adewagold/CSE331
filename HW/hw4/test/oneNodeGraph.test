# Tests if a node can be added to the graph

# Create a graph
CreateGraph one_node_graph

# Add a node
AddNode one_node_graph a

# Print the nodes in the graph and the outgoing edges from a
ListNodes one_node_graph
ListChildren one_node_graph a