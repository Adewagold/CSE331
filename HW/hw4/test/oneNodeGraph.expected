# Tests if a node can be added to the graph

# Create a graph
created graph one_node_graph

# Add a node
added node a to one_node_graph

# Print the nodes in the graph and the outgoing edges from a
one_node_graph contains: a
the children of a in one_node_graph are: